packages/protocol/test/TestBase.sol

```diff
@@ -47,6 +48,11 @@ abstract contract TestBase is Test {
     address internal Xavier = getRandomAddress();
     address internal Yasmine = getRandomAddress();
     address internal Zachary = getRandomAddress();
+    address internal SGX_X_0 = vm.addr(0x4);
```

Comment:
@adaki2004 please try to avoid adding test-specific code into TaikoL1TestBase, this base class is supposed to be clean and decoupled from actual tests